torch/_C/__init__.pyi.in

```diff
@@ -308,14 +309,20 @@ def _jit_pass_onnx_eliminate_unused_items(graph: Graph, paramsDict: Dict[str, IV
 def _jit_pass_onnx_cast_all_constant_to_floating(graph: Graph) -> None: ...
 def _jit_pass_filter_non_tensor_arguments(params: Dict[str, IValue]) -> Dict[str, Tensor]: ...
 def _jit_decay_packed_param_input_types(graph: Graph) -> None: ...
-def _jit_pass_onnx_node_shape_type_inference(n: Node, paramsDict: Dict[str, IValue], opset_version: _int) -> None: ...
-def _jit_onnx_convert_pattern_from_subblock(block: Block, n: Node, env: Dict[Value, Value]) -> List[Value]: ...
+def _jit_pass_onnx_node_shape_type_inference(n: Node, paramsDict: Dict[str, IValue], opset_version: _int,
+    static_input_shape: _bool) -> None: ...
+def _jit_onnx_convert_pattern_from_subblock(block: Block, n: Node, env: Dict[Value, Value], static_input_shape: _bool) -> List[Value]: ...
 def _jit_pass_onnx_block(
     old_block: Block,
     new_block: Block,
     operator_export_type: _onnx.OperatorExportTypes,
-    env: Dict[Value, Value]
-) -> None: ...
+    env: Dict[Value, Value],
+    static_input_shape: _bool,
+    paramsDict: Dict[str, IValue],
+    opset_version: _int,
+    enable_const_folding: _bool,
```

Comment:
This flag is actually in `_jit_pass_onnx_block` -- it is because this PR originally has a bug (will reply in your other comments).
Now it is not needed.